src/context/WalletProvider/NativeWallet/components/NativeImportKeystore.tsx

```diff
@@ -27,7 +27,7 @@ import type { NativeWalletValues } from '../types'
 const hoverSx = { borderColor: 'blue.500' }
 
 // TODO(gomes): use https://www.chakra-ui.com/docs/components/file-upload if/when we migrate to chakra@3
-const FileUpload = ({ onFileSelect }: { onFileSelect: (file: File) => void }) => {
+export const FileUpload = ({ onFileSelect }: { onFileSelect: (file: File) => void }) => {
```

Comment:
nitpick: since this is now shared, would be better off living in its own home

Reply:
Exported it in [b0dae5c](https://github.com/shapeshift/web/pull/8848/commits/b0dae5cef385e8876b2434989d8da73dca1c2b25)